pkg/resource/azurerm/metadata.go

```diff
@@ -0,0 +1,6 @@
+package azurerm
+
+import "github.com/cloudskiff/driftctl/pkg/resource"
+
+func InitResourcesMetadata(resourceSchemaRepository resource.SchemaRepositoryInterface) {
+}
```

Comment:
I think we can remove that function?

Reply:
It's for future azure resources I guess and it will provide our contributors a way to just add corresponding metadata methods inside this function as per our docs.

Reply:
yeah I've confused it with a single resource metadata